src/gentropy/config.py

```diff
@@ -38,6 +38,9 @@ class ColocalisationConfig(StepConfig):
     credible_set_path: str = MISSING
     coloc_path: str = MISSING
     colocalisation_method: str = MISSING
+    priorc1: float | None = MISSING
```

Comment:
It can be either Optional parameter (float | None) or required parameter `priroc1: float = MISSING`

Reply:
Done